trainingRN/src/constants/routes.js

```diff
@@ -0,0 +1,2 @@
+export const HOME = 'Home';
+export const BOOK_LIST = 'BookList' 
```

Comment:
I'd export this as a whole, single object, so we can use
```
import routes from './constants/routes`
...
routes.HOME
```
I think this way we denote that these constants are indeed routes. What do you think ?